scripts/xcm-asset-registrator.ts

```diff
@@ -80,13 +82,18 @@ async function main () {
         ));
     }
 
-    const batchTx = api.tx.utility.batchAll(registerTxs);
+    let toPropose = api.tx.utility.batchAll(registerTxs);
+
+    if (args['at-block']) {
+        const call = { Value: toPropose };
+        toPropose = api.tx.scheduler.schedule(args["at-block"], null, 0, call);
+    }
```

Comment:
```
    const batchCall = api.tx.utility.batchAll(registerTxs);
    const toPropose = args['at-block'] ? 
      api.tx.scheduler.schedule(args["at-block"], null, 0, call) :
      batchCall;
```
I think this looks a bit better in the sense there is no "variables", only "const" but up to you if you find it more readable or not